common-service-core/src/main/java/com/ca/mfaas/message/api/ApiMessage.java

```diff
@@ -34,37 +62,49 @@
      *
      * Example: "PFI0031"
      */
-    String getMessageNumber();
+    public String getMessageNumber() {
+        return messageNumber;
+    }
 
     /**
      * Readable message in US English. This field is required. It should be a sentence starting with a capital letter
      * and ending with a full stop (.).
      */
-    String getMessageContent();
+    public String getMessageContent() {
+        return messageContent;
+    }
 
     /**
      * Supplements the messageContent field, supplying more information about why the message is present.
      * This field is optional.
      */
-    String getMessageAction();
+    public String getMessageAction() {
+        return messageAction;
+    }
 
     /**
      * Recommendation of the actions to take in response to the message.
      * This field is optional.
      */
-    String getMessageReason();
+    public String getMessageReason() {
+        return messageReason;
+    }
 
     /**
      * Optional unique key describing the reason of the error.
      * It should be a dot delimited string "com.ca.service[.subservice].detail".
      * The purpose of this field is to enable UI to show a meaningful and localized error message.
      */
-    String getMessageKey();
+    public String getMessageKey() {
+        return messageKey;
+    }
 
     /**
      * Optional error message parameters. Used for formatting of localized messages.
      */
-    List<Object> getMessageParameters();
+    public List<T> getMessageParameters() {
+        return messageParameters;
+    }
 
     /**
      * Optional unique ID of the message instance. Useful for finding of the message in the logs.
```

Comment:
"Finds (or "locates/identifies") the logs messages"

Reply:
Actually, this method doesn't find log messages. It points to unique id for messages which developers can find the logs easily. But it doesn't identify any logs. That why I would keep "Useful for..."  beginning for the sentence. Something like that:
"Useful for identifying the message in the logs"
It is ok?

Reply:
then "Used for identifying the message in the logs". Also, this statement is correct if only 1 message (some specific message). If there are more than 1 possible message, write "Used for identifying messages in the logs"